example/partials/example/_editor.yaml

```diff
@@ -1,4 +1,5 @@
-label: Example Partial
+label: Example Partial ascascas123
```

Comment:
Did you want to make this just a longer string instead of random characters?

Reply:
Oops. :) This wasn't meant to make it into the PR.